lib/perl/Genome/Model/ClinSeq/Command/MakeCircosPlot.pm

```diff
@@ -963,43 +972,63 @@ EOS
 
     ### Tier1 SNVs and INDELs
 
-    #SNV
-    my $snv_file = Genome::Sys->read_file("$output_directory/raw/snvs.hq.tier1.v1.annotated.compact.tsv");
     my $snv_fh   = Genome::Sys->open_file_for_writing("$output_directory/data/snvs.hq.tier1.v1.annotated.compact.tsv");
-    while ($snv_file =~ /(\S+):(\d+)-(\d+)\s+\S+\s+(\S+)\s+\w+\s+(\S+)\s+(\S+)\s+(\S+).*/g) {
-
-        $genes_noAmpDel{$4} = "hs$1\t$2\t$3";
-        $genes_AmpDel{$4}   = "hs$1\t$2\t$3";
-        print $snv_fh "hs$1 $2 $3 0.5 fill_color=black\n";
-
-#        switch($5){
-#            case "nonsense"            {print $snv_fh "fill_color=goldenrod\n"}
-#            case "missense"            {print $snv_fh "fill_color=blue\n"}
-#            case "silent"            {print $snv_fh "fill_color=green\n"}
-#            case "splice_site"        {print $snv_fh "fill_color=black\n"}
-#            case "rna"                {print $snv_fh "fill_color=purple\n"}
-#            case "nonstop"            {print $snv_fh "fill_color=black\n"}
-#        }
-
-    }
-    $snv_fh->close;
-
-    #Indel
-    my $indel_file = Genome::Sys->read_file("$output_directory/raw/indels.hq.tier1.v1.annotated.compact.tsv");
     my $indel_fh =
         Genome::Sys->open_file_for_writing("$output_directory/data/indels.hq.tier1.v1.annotated.compact.tsv");
-    my $color;
-    while ($indel_file =~ /(\S+):(\d+)-(\d+)\s+\S+\s+(\S+)\s+\w+\s+(\S+)\s+(\S+)\s+(\S+).*/g) {
-
-        $genes_noAmpDel{$4} = "hs$1\t$2\t$3";
-        $genes_AmpDel{$4}   = "hs$1\t$2\t$3";
-        if   ($6 eq "-") {$color = "red";}
-        else             {$color = "blue";}
-
-        print $indel_fh "hs$1 $2 $3 0.5 fill_color=$color\n";
+    #SNV
+    unless ($self->annotated_variants_tsv ) {
+        my $snv_file = Genome::Sys->read_file("$output_directory/raw/snvs.hq.tier1.v1.annotated.compact.tsv");
+        while ($snv_file =~ /(\S+):(\d+)-(\d+)\s+\S+\s+(\S+)\s+\w+\s+(\S+)\s+(\S+)\s+(\S+).*/g) {
+            $genes_noAmpDel{$4}="hs$1\t$2\t$3";
+            $genes_AmpDel{$4}="hs$1\t$2\t$3";
+            print $snv_fh "hs$1 $2 $3 0.5 fill_color=black\n";
+            #        switch($5){
+            #            case "nonsense"            {print $snv_fh "fill_color=goldenrod\n"}
+            #            case "missense"            {print $snv_fh "fill_color=blue\n"}
+            #            case "silent"            {print $snv_fh "fill_color=green\n"}
+            #            case "splice_site"        {print $snv_fh "fill_color=black\n"}
+            #            case "rna"                {print $snv_fh "fill_color=purple\n"}
+            #            case "nonstop"            {print $snv_fh "fill_color=black\n"}
+            #        }
```

Comment:
I'd say drop this since we can always look at git history later. @bainscou to confirm!